classes/CartRule.php

```diff
@@ -1699,8 +1700,8 @@ public static function autoRemoveFromCart(Context $context = null, bool $useOrde
         }
 
         static $errors = [];
-        foreach ($context->cart->getCartRules(CartRule::FILTER_ACTION_ALL, true, $useOrderPrice) as $cart_rule) {
-            if ($error = $cart_rule['obj']->checkValidity($context, true, true, true, $useOrderPrice)) {
+        foreach ($context->cart->getCartRules(CartRule::FILTER_ACTION_ALL, $autoAdd, $useOrderPrice) as $cart_rule) {
+            if ($error = $cart_rule['obj']->checkValidity($context, true, true, true, $useOrderPrice, true)) {
```

Comment:
Hello @pmartin-webimpacto ,

Is the last parameter useful?

I don't see any modification for the checkValidity method.